### PR TITLE
Update to 3.3.0

### PR DESCRIPTION
**Pull request by nicolas377**
This is version 3.3.0, featuring multiple new infrastructure systems and a new script called keyboard mapping.

Changes:
<ul>
  <li><b>Add support for multiple key press keybinds.</b></li>
  <li><b>Moved all the code to a monorepo.</b> #81</li>
  <li><i>Switch script optimization to webpack.</i> #120</li>
  <li><i>Style type names to UpperCamelCase.</i> #122</li>
  <li>
    <ins>FMC uses data from previous routes when fields are empty.</ins> #87
  </li>
</ul>
This update includes functional changes through the addition of a new script.
<br><br>

Please fill in this template.

- [x] Use a meaningful title for the pull request.
- [x] Test the change. (Compile and use the built extension.)
- [x] Run `npm run build`.
- [x] Run `npm test`.
- [x] Pass testing.

### CODE REVIEW

**Comment by nicolas377** [comment]
3.3.0 has been posted to testers, and we're expecting a go to merge within 48 hours.

File: .eslintrc.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
 {
   "env": {
-    "browser": true,
-    "es2021": true,
-    "webextensions": true
+    "es2021": true
+  },
+  "globals": {
+    "requirejs": false
```
Comment (Guy-Adler) [inline .eslintrc.json:6]: we don't need requirejs at all, this can be removed

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      required: true
+    attributes:
+      label: Operating System
+      description: The OS that you were running when you discovered the problem. Include the version number.
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: Browser
+      description: The OS that you were running when you discovered the problem. Include the version number
```
Comment (Guy-Adler) [inline .github/ISSUE_TEMPLATE/bug.yml:46]: The sentence should be changed to
> The browser that you were running when you discovered the problem. Include the version number.

File: SECURITY.md [hunk 3]
```diff
@@ -4,8 +4,8 @@
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 3.1.x   | :white_check_mark: |
-| < 3.1   | :x:                |
+| 3.2.x   | :white_check_mark: |
```
Comment (Guy-Adler) [inline SECURITY.md:7]: need to be updated to 3.3.x

**Review by Guy-Adler — CHANGES_REQUESTED** [review]
Changes requested in comments.

**Review by Guy-Adler — DISMISSED** [review]
Looks good to me
we might want to remove the @ts-ignore in fmc/source/ui/ViewModel.ts#L64 and fmc/source/waypoints.ts#L402, but I think it can be done in 3.4.0 to not delay this release.

**Review by Guy-Adler — APPROVED** [review]

**Pull request closed without merging**